Corona-Warn-App/schemas/de.rki.coronawarnapp.eventregistration.storage.TraceLocationDatabase/1.json

```diff
@@ -62,6 +68,12 @@
             "affinity": "INTEGER",
             "notNull": false
           },
+          {
+            "fieldPath": "byteRepresentation",
+            "columnName": "byteRepresentation",
+            "affinity": "BLOB",
+            "notNull": true
+          },
           {
             "fieldPath": "signature",
             "columnName": "signature",
```

Comment:
should not renamed to `signatureBase64` too? as in [here](https://github.com/corona-warn-app/cwa-app-android/pull/2640/files#diff-f55fc45103a717bbbd6613c2435d745790edbb95840f4eacbc96feb685da9654R170)

Reply:
you're absolutely right, we should do it the same way!